data/vendor-customer-support.ini

```diff
@@ -0,0 +1,8 @@
+[Customer Support]
```

Comment:
I don't think we want the contact info in Transifex.  @mcatanzaro is correct that we do not maintain es_MX vs. es_GT translations.  And, eventually, I suspect we may need other rules for selecting the appropriate contact info.  For now, the ini file works for me as is.